build.gradle

```diff
@@ -13,4 +13,8 @@ repositories {
 dependencies {
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.7'
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.24'
+    annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.24'
+    testCompileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.24'
+    testAnnotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.24'
```

Comment:
I typically add the Lombok gradle plugin for this (it would also be a good time to migrate the old `apply plugin` format to the `plugins` format. What do you think about this instead?

```groovy
plugins {
    id 'java'
    id 'maven-publish'
    id 'io.freefair.lombok' version '6.6'
}
```

```suggestion
```

Reply:
I know maven much better than I do gradle, so whatever you think is best! If that works then LGTM!

I did briefly try the plugin but gradle couldn't find it. Might have been a version problem

Reply:
That's most likely the issue, the version of Gradle being used was 4.x which is 3 major versions old. I updated it on the master branch to the latest version so it should be good going forward. 👍